package.json

```diff
@@ -47,6 +47,10 @@
     "create-react-class": "15.5.3",
     "creditcards": "2.1.2",
     "cross-env": "4.0.0",
+    "d3-array": "1.2.0",
```

Comment:
@blowery Checked and am on `3.10.10`

![screen shot 2017-06-08 at 5 05 29 pm](https://user-images.githubusercontent.com/1160732/26935624-ba051124-4c6c-11e7-9162-e39df1d40277.png)